bluemix/configuration/persistor.go

```diff
@@ -44,14 +47,26 @@ func (dp DiskPersistor) Load(data DataInterface) error {
 		return err
 	}
 
-	if err != nil {
-		err = dp.write(data)
+	if err != nil { // strange: requiring an error (to allow write attempt to continue), as long as it is not a permission error
```

Comment:
Strange code. 
Requiring a non-permission error to actually carry out the writing to the config. Weird!
Please see my OP for my thoughts on why this is benign despite being pretty obviously counterintuitive.

Reply:
It is strange but the docs strongly recommend using **TryLock** instead due to the potential blocking issue. Can we try that and see if it resolves this issue

> Lock is a blocking call to try and take an exclusive file lock. It will wait until it is able to obtain the exclusive file lock. It's recommended that TryLock() be used over this function. This function may block the ability to query the current Locked() or RLocked() status due to a RW-mutex lock. ([doc](https://pkg.go.dev/github.com/gofrs/flock#Flock.Lock))